docs/wiki/01 - STUDY PARAMETERS SETUP/01 - Study Variables/05 - Timepoints.md

```diff
@@ -0,0 +1,52 @@
+# Timepoints
+
+## Overview
+**Timepoints** are a list of possible instances in time where a candidate can visit or contact a site with the purpose of undergoing any form of data collection.
+
+**Timepoint** and **Visit** are terms used interchangeably in LORIS (the preferred term is **Timepoint**). **Timepoints** are defined in the `Visit_Windows` and `visit` tables of the database as well as the `<visitLabel>` tags in the `config.xml`.
+
+## Adding Timepoint Options
+
+### Front End
+ _not yet available_
+
+### SQL
+
+The `Visit_Windows` table lists the timepoints in the study and adds some study-specific age ranges defined for each timepoint. This table must be populated with visit labels for the proper functioning of LORIS. To populate the table with visit labels insert study-specific information as follows:
+
+```sql
+INSERT INTO Visit_Windows (Visit_label,  WindowMinDays, WindowMaxDays, OptimumMinDays, OptimumMaxDays, WindowMidpointDays) VALUES ('V1', 0, 100, 40, 60, 50);
+```
+
+If age is not a critical factor in study timepoint scheduling, define `Min` and `Max` values as `NULL`.
```

Comment:
lol. Here they should be set to `NULL` and in the battery table they should be set to `0`. Very consistent ;).